releasenotes/notes/cca_in_ad_on-5ec511f4f574337b.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    The ``logs_config.cca_in_ad`` feature flag now defaults to true.  This
```

Comment:
is this supposed to have double backticks?

Reply:
Yes, it's reStructuredText.